configure/ingress-nginx/mandatory.yaml

```diff
@@ -1,63 +1,56 @@
+
 apiVersion: v1
 kind: Namespace
 metadata:
   name: ingress-nginx
   labels:
     app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/instance: ingress-nginx
 
 ---
-
-kind: ConfigMap
+# Source: ingress-nginx/templates/controller-serviceaccount.yaml
 apiVersion: v1
+kind: ServiceAccount
 metadata:
-  name: nginx-configuration
-  namespace: ingress-nginx
   labels:
+    helm.sh/chart: ingress-nginx-3.19.0
     app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
-
----
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: tcp-services
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/version: 0.43.0
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/component: controller
+  name: ingress-nginx
   namespace: ingress-nginx
-  labels:
-    app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
-
 ---
-kind: ConfigMap
+# Source: ingress-nginx/templates/controller-configmap.yaml
 apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: udp-services
-  namespace: ingress-nginx
   labels:
+    helm.sh/chart: ingress-nginx-3.19.0
     app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
-
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: nginx-ingress-serviceaccount
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/version: 0.43.0
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/component: controller
+  name: ingress-nginx-controller
   namespace: ingress-nginx
-  labels:
-    app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
-
+data:
 ---
-apiVersion: rbac.authorization.k8s.io/v1beta1
+# Source: ingress-nginx/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: nginx-ingress-clusterrole
   labels:
+    helm.sh/chart: ingress-nginx-3.19.0
     app.kubernetes.io/name: ingress-nginx
-    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/version: 0.43.0
+    app.kubernetes.io/managed-by: Helm
```

Comment:
```suggestion
```

I understand this comes from Helm, but its not managed by it in our deployment.

Reply:
I will not comment one very instance to keep the discussion here. I would remove all `managed-by` and `helm.sh/chart` references.